packages/nodejs/src/__tests__/config.test.ts

```diff
@@ -130,10 +130,10 @@ describe("Configuration", () => {
 
     describe("with logPath option with file specified", () => {
       beforeEach(() => {
-        config = new Configuration({ logPath: "/other_path/appsignal.log" })
+        config = new Configuration({ logPath: "/other_path/foo.log" })
       })
 
-      it("uses the overwritten path", () => {
+      it("uses the overwritten path but changes the file name", () => {
```

Comment:
Can you also add an assertion here that the deprecation warning is being printed to `console.warn`?